src/seedu/addressbook/data/person/Address.java

```diff
@@ -2,16 +2,27 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
 
+import seedu.addressbook.data.person.address.Block;
+import seedu.addressbook.data.person.address.PostalCode;
+import seedu.addressbook.data.person.address.Street;
+import seedu.addressbook.data.person.address.Unit;
+
 /**
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String EXAMPLE = "123, Hougang, #12-12, 123456";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in "
+    		+ "the format of BLOCK, STREET, UNIT, POSTAL CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
+    private final Block block;
```

Comment:
Why is this final? Do you have setter methods to change the values if they change later on. Check it